recipes/genoboo/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -exo pipefail
+
+outdir=${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}
+mkdir -p $outdir $PREFIX/bin
+
+# Fix architecture value
+export ARCH=$(uname -m)
+
+# Need a home for osx
+export HOME="/tmp/"
+
+# Install meteor here (instead of in npm run bundle) because we need to patch the install script
+curl "https://install.meteor.com/?release=2.8.0" > meteor.sh
+chmod a+x meteor.sh
+sed -i.bak 's|PREFIX=|#PREFIX=|' meteor.sh
+./meteor.sh
+
+npm install --unsafe-perm
+
+export PATH=$PATH:"$HOME/.meteor"
```

Comment:
No idea about the general strategy, but intuitively I would guess that it would be better to have this in conda as well?

Reply:
OK. I see this is installing a fork .. https://github.com/bioconda/bioconda-recipes/tree/master/recipes/genenotebook  .. is this a good idea? Do you know if the fork will stay?

Reply:
OK. Cool with me. Maybe just open a PR from your master to their master. Then they can merge any time and are aware of your fork.